### ISSUE TITLE
As a user I want to define timetable settings for exceptional time periods

### ISSUE THREAD

**Issue by ellinooraikaheimo** [HSLdevcom/jore4#955]
User can define exceptional time periods in the global timetable settings view (toistuvat erityispäivät, satunnaiset erityispäivät, paikkakuntakohtaiset jaksot)

User can enter the global timetable settings from the timetables main page.

![Image](https://user-images.githubusercontent.com/77675592/188847697-ab20e6aa-fee3-497a-a8ca-f8821092a67f.png)

User can save once ready. The made settings will be visualized in the Lines timetable view presenting the day types.

Blocked by #681, creating the data model for timetables.

Note: Defining settings for Seasons and day types are split to another ticket. 

Figma: https://www.figma.com/file/ImSTkCqQn0nhVUtMcUm41P/JORE-4.0-UX?node-id=3238%3A255458
